README.md

```diff
@@ -1,7 +1,6 @@
 # laminas-development-mode
 
-[![Build Status](https://travis-ci.com/laminas/laminas-development-mode.svg?branch=master)](https://travis-ci.com/laminas/laminas-development-mode)
-[![Coverage Status](https://coveralls.io/repos/github/laminas/laminas-development-mode/badge.svg?branch=master)](https://coveralls.io/github/laminas/laminas-development-mode?branch=master)
+[![Build Status](https://github.com/laminas/laminas-development-mode/workflows/Continuous%20Integration/badge.svg)](https://github.com/laminas/laminas-development-mode/actions?query=workflow%3A"Continuous+Integration")
```

Comment:
Please use the workflow filename instead of its name as described here: 
https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-workflow-file-name